experiment/autobumper/main.go

```diff
@@ -17,167 +17,40 @@ limitations under the License.
-	logrus.Infof("Changing working directory to %s...", d)
-	return os.Chdir(d)
-}
-
-type options struct {
-	githubLogin string
-	githubToken string
-	gitName     string
-	gitEmail    string
-}
-
-func parseOptions() options {
-	var o options
-	flag.StringVar(&o.githubLogin, "github-login", "", "The GitHub username to use.")
-	flag.StringVar(&o.githubToken, "github-token", "", "The path to the GitHub token file.")
-	flag.StringVar(&o.gitName, "git-name", "", "The name to use on the git commit. Requires --git-email. If not specified, uses values from the user associated with the access token.")
-	flag.StringVar(&o.gitEmail, "git-email", "", "The email to use on the git commit. Requires --git-name. If not specified, uses values from the user associated with the access token.")
+func parseOptions() *bumper.Options {
+	var o bumper.Options
+	flag.StringVar(&o.GitHubToken, "github-token", "", "The path to the GitHub token file.")
+	flag.StringVar(&o.GitHubLogin, "github-login", "", "The GitHub username to use. If not specified, uses values from the user associated with the access token.")
+	flag.StringVar(&o.GitName, "git-name", "", "The name to use on the git commit. Requires --git-email. If not specified, uses values from the user associated with the access token.")
+	flag.StringVar(&o.GitEmail, "git-email", "", "The email to use on the git commit. Requires --git-name. If not specified, uses values from the user associated with the access token.")
+	flag.StringVar(&o.GitHubOrg, "github-org", "", "The GitHub org name where the autobump PR will be created. Must not be empty when --create-pull-request is not false.")
+	flag.StringVar(&o.GitHubRepo, "github-repo", "", "The GitHub repo name where the autobump PR will be created. Must not be empty when --create-pull-request is not false.")
+	flag.StringVar(&o.OncallAddress, "oncall-address", "", "The oncall address where we can get the JSON file that stores the current oncall information.")
+
+	flag.BoolVar(&o.BumpProwImages, "bump-prow-images", false, "Whether to bump up version of images in gcr.io/k8s-prow/.")
+	flag.BoolVar(&o.BumpTestImages, "bump-test-images", false, "Whether to bump up version of images in gcr.io/k8s-testimages/.")
+	flag.StringVar(&o.TargetVersion, "target-version", "", "The target version to bump images version to, which can be one of latest, upstream, upstream-staging and vYYYYMMDD-deadbeef.")
+
+	flag.Var(&o.IncludedConfigPaths, "include-config-paths", "The config paths to be included in this bump, in which only .yaml files will be considered. By default all files are included.")
+	flag.Var(&o.ExcludedConfigPaths, "exclude-config-paths", "The config paths to be excluded in this bump, in which only .yaml files will be considered.")
+	flag.Var(&o.ExtraFiles, "extra-files", "The extra non-yaml file to be considered in this bump.")
+
+	flag.BoolVar(&o.CreatePullRequest, "create-pull-request", true, "Whether to create a pull request for this bump.")
```

Comment:
skip-pull-request makes a bit more sense to me, as the default can be false

Reply:
Makes sense. Done.